src/main/java/frc/robot/Robot.java

```diff
@@ -81,6 +86,8 @@ public void robotPeriodic() {
     CommandScheduler.getInstance().run();
     SmartDashboard.putNumber("thng", ShooterFlywheel.getInstance().getLeftFlywheelSpeed());
     SmartDashboard.putNumber("thng2", ShooterFlywheel.getInstance().getRightFlywheelSpeed());
+    SmartDashboard.putNumber("climbele", ShooterElevator.getInstance().getPosition()*ShooterElevator.getInstance().gearing);
```

Comment:
also what does "gearing" mean? do you mean a gear ratio? either way if this is a constant it should be `TITLE_CASE` and a `public static final double`